codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 90%
```

Comment:
90% is arbitrary but at least now if we want to lower the threshold it will be explicit.

Reply:
i like that